run/src/test/scala/sbt/ForkTest.scala

```diff
@@ -14,9 +14,9 @@ object ForkTest extends Properties("Fork") {
   final val MaximumClasspathLength = 100000
 
   lazy val genOptionName = frequency((9, Some("-cp")), (9, Some("-classpath")), (1, None))
-  lazy val pathElement = listOf1(alphaNumChar).map(_.mkString)
-  lazy val path = listOf1(pathElement).map(_.mkString(File.separator))
-  lazy val genRelClasspath = listOf1(path)
+  lazy val pathElement = nonEmptyListOf(alphaNumChar).map(_.mkString)
+  lazy val path = nonEmptyListOf(pathElement).map(_.mkString(File.separator))
+  lazy val genRelClasspath = nonEmptyListOf(path)
```

Comment:
`Gen.listOf1` from Scalacheck has been deprecated and replaced by `Gen.nonEmptyListOf`.